client/mysqltest.cc

```diff
@@ -6170,6 +6171,22 @@ enum block_op find_operand(const char *start)
  return ILLEG_OP;
 }
 
+void do_break(struct st_command* command)
```

Comment:
add a comment before the function, see how it's done before every other do_xxx() function.